### ISSUE TITLE
Add an option to set font "bold" font style

### ISSUE THREAD

**Issue by michaelfresco** [jupyter/qtconsole#247]
I was hoping to run QtConsole with my favorite font, Monaco 13 point, in **bold**.
Some fonts look a lot better in bold. **Monaco** is one of these. 

I hope you can add a option to start qtconsole with Monaco bold 13.

**Comment by ccordoba12** [comment]
You can change the font family and size by setting the option options

     JupyterWidget.font_family
     JupyterWidget.font_size

This document describes how to set options for different Jupyter applications:

http://jupyter.readthedocs.io/en/latest/projects/config.html

**Comment by michaelfresco** [comment]
I found a solution: Install a custom version of Monaco which has a bold style. 
[Link to Monaco Bold project](https://github.com/vjpr/monaco-bold). This gives Jupyter QTConsole pretty much the same look as Sublime.

If I had to make a guess why Sublime renders Monaco differently than Jupyter QTConsole, I would say that Sublime follows the MacOS anti-aliasing rules. So fonts like Monaco will get bigger, and bolder to improve readability. I guess Jupyter does not anti-alias by default.

Regardless, I found a workaround by installing a custom version of Monaco.

![screen shot 2017-11-27 at 12 56 18 pm](https://user-images.githubusercontent.com/15065392/33265547-685df5ec-d372-11e7-97c2-54d56b0f6eb8.png)
